package.json

```diff
@@ -59,7 +59,7 @@
 		"@skyra/char": "^1.0.1",
 		"@skyra/decorators": "discordjs",
 		"canvas": "^2.6.1",
-		"canvas-constructor": "^3.2.0",
+		"canvas-constructor": "kyranet/canvas-constructor#build",
```

Comment:
This should become the NPM published version before we merge this PR

Reply:
Resolved in b5ef2bb.